modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -1298,8 +1298,7 @@ definition {
 	@description = "LPS-89416 - Verify that updating a SLA will recalculate time for all instances In Progress"
 	@priority = 5
 	test UpdatingSLARecalculateTimeInProgressInstances {
-		property portal.upstream = "quarantine";
-		property test.name.skip.portal.instance = "WorkflowMetricsSLA#UpdatingSLARecalculateTimeInProgressInstances";
```

Comment:
Hey @victorhcunha, don't remove this property:` property test.name.skip.portal.instance`

This property is used to pass on CI. Some things on CI are not possible to use in Virtual Instance (where the CI ran). So, we used this property to run in the normal instance.

Thanks!